db-scripts/createdb.sql

```diff
@@ -193,16 +193,16 @@ create table if not exists event_companies(
    company_id integer not null,
    PRIMARY KEY(event_id, company_id),
    FOREIGN KEY(event_id) REFERENCES events(id),
```

Comment:
let's also add on delete cascade for this fk

Reply:
done